perf-dash.k8s.io/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: perfdash
+  namespace: perfdash
+  labels:
+    app: perfdash
+spec:
+  selector:
+    matchLabels:
+      app: perfdash
+  template:
+    metadata:
+      labels:
+        app: perfdash
+    spec:
+      containers:
+      - name: perfdash
+        image: gcr.io/k8s-testimages/perfdash:2.21
```

Comment:
NIT: Yesterday we bumped perfdash to 2.22

Reply:
I'll bump it too in a minute

Reply:
Done